src/scripts/training/lightgbm_python/spec.yaml

```diff
@@ -160,6 +160,12 @@ inputs:
     description: "For benchmark analysis, provide as a json dictionary (ex: {\"foo\":\"bar\"}) anything that will be added as tags to the job"
     optional: true
 
+  model_filename:
+    type: String
+    description: "Name of the model output file"
+    optional: true
```

Comment:
could you set optional:false ? you can't have a default value for something that is optional (or else, what's the point of a default value?) I know we might have that here and there in other spec, but we should avoid it.

set it to optional:false and keep your default as it